<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Bharani" failures="3" tests="21" name="Test" time="42.086" errors="0" timestamp="2025-09-20T11:07:30 MYT">
  <testcase classname="petstore.testcases.UserDataDriventests" name="testPostUser" time="3.552"/>
  <testcase classname="petstore.testcases.UserDataDriventests" name="testPostUser" time="1.522"/>
  <testcase classname="petstore.testcases.UserDataDriventests" name="testPostUser" time="1.632"/>
  <testcase classname="petstore.testcases.UserDataDriventests" name="testDeleteUserByName" time="1.591"/>
  <testcase classname="petstore.testcases.UserDataDriventests" name="testDeleteUserByName" time="1.53"/>
  <testcase classname="petstore.testcases.UserDataDriventests" name="testDeleteUserByName" time="1.533"/>
  <testcase classname="petstore.testcases.StoreDataDriventests" name="testPostStoreOrder_DD" time="1.612"/>
  <testcase classname="petstore.testcases.StoreDataDriventests" name="testPostStoreOrder_DD" time="1.633"/>
  <testcase classname="petstore.testcases.StoreDataDriventests" name="testPostStoreOrder_DD" time="1.525"/>
  <testcase classname="petstore.testcases.StoreDataDriventests" name="testDeleteStoreOrderByID_DD" time="1.565"/>
  <testcase classname="petstore.testcases.StoreDataDriventests" name="testDeleteStoreOrderByID_DD" time="1.535"/>
  <testcase classname="petstore.testcases.StoreDataDriventests" name="testDeleteStoreOrderByID_DD" time="1.534"/>
  <testcase classname="petstore.testcases.StoreDataDriventests" name="testGetStoreOrderByID_DD_Negative" time="2.427"/>
  <testcase classname="petstore.testcases.StoreDataDriventests" name="testGetStoreOrderByID_DD_Negative" time="1.643">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
JSON path message doesn&amp;apos;t match.
Expected: Order not found
  Actual: null
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path message doesn't match.
Expected: Order not found
  Actual: null

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:58)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:270)
at io.restassured.specification.ResponseSpecification$body$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:117)
at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:244)
at petstore.testcases.StoreDataDriventests.testGetStoreOrderByID_DD_Negative(StoreDataDriventests.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testGetStoreOrderByID_DD_Negative -->
  <testcase classname="petstore.testcases.StoreDataDriventests" name="testGetStoreOrderByID_DD_Negative" time="1.598">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
JSON path message doesn&amp;apos;t match.
Expected: Order not found
  Actual: null
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path message doesn't match.
Expected: Order not found
  Actual: null

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:696)
at io.restassured.internal.ResponseSpecificationImpl.this$2$validateResponseIfRequired(ResponseSpecificationImpl.groovy)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:198)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:62)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:270)
at io.restassured.specification.ResponseSpecification$body$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:117)
at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:244)
at petstore.testcases.StoreDataDriventests.testGetStoreOrderByID_DD_Negative(StoreDataDriventests.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testGetStoreOrderByID_DD_Negative -->
  <testcase classname="petstore.testcases.Usertests" name="testCreateUser" time="3.619"/>
  <testcase classname="petstore.testcases.Storetests" name="testPostStoreOrder" time="4.012"/>
  <testcase classname="petstore.testcases.Storetests" name="testGetStoreOrder" time="1.638">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at petstore.testcases.Storetests.testGetStoreOrder(Storetests.java:84)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testGetStoreOrder -->
  <testcase classname="petstore.testcases.Storetests" name="testDeleteStoreOrder" time="1.577"/>
  <testcase classname="petstore.testcases.AuthenticationTests" name="testAuthentication" time="2.034"/>
  <testcase classname="petstore.testcases.AuthenticationTests" name="testApiKey" time="0.179"/>
</testsuite> <!-- Test -->
